src/services/api.ts

```diff
@@ -0,0 +1,44 @@
+import type { Fruit } from '../components/common.types'
+import axios from 'axios';
+
+//response from API is and array of objects so we need to specify its type.
```

Comment:
i know these comments are info for the future, but keep'em short, i've seen a lot. the "con axios" one can be removed